README.md

```diff
@@ -276,6 +276,16 @@ Here are all the available options:
     <td>Preserve value at Move Up or Down.(No value is selected automatically upon deletion.)</td>
     <td><code>true</code></td>
   </tr>
+  <tr>
+    <td>max_depth</td>
+    <td>Max depth of an object that have to be rendered. The missing of this option could cause "maximum call stack size exceeded" in case of recursive object properties</td>
```

Comment:
```suggestion
    <td>Max depth of the nested properties to be rendered of provided json schema. The missing of this option could cause "maximum call stack size exceeded" in case of object properties with circular references</td>
```